exercises/concept/resistor-color/.meta/exemplar.rs

```diff
@@ -0,0 +1,30 @@
+use enum_iterator::IntoEnumIterator;
+
+#[derive(Debug, IntoEnumIterator, PartialEq)]
+pub enum ResistorColor {
+    Black = 0,
+    Brown = 1,
+    Red = 2,
+    Orange = 3,
+    Yellow = 4,
+    Green = 5,
+    Blue = 6,
+    Violet = 7,
+    Grey = 8,
+    White = 9,
+}
+
+pub fn color_to_value(color: ResistorColor) -> usize {
+    color as usize
+}
+
+pub fn value_to_color_string(value: usize) -> String {
+    ResistorColor::into_enum_iter()
+        .nth(value)
+        .map(|color| format!("{:?}", color))
+        .unwrap_or_else(|| "value out of range".into())
+}
```

Comment:
Why not retain the `IntEnum`-based implementation? I'd expect that to be able to produce a faster lookup than iterating through the entire list. I don't think it's a problem for the concept introducing external dependencies to have two different external dependencies.

Reply:
I thought this was what you meant by 

> it should use the enum-iterator crate

To be honest, I'm not sure whether the two crates would interact nicely.

Reply:
Ok, sure, I'll try!